pkg/apis/management.cattle.io/v3/catalog_types.go

```diff
@@ -35,11 +35,13 @@ type CatalogSpec struct {
 }
 
 type CatalogStatus struct {
-	LastRefreshTimestamp string `json:"lastRefreshTimestamp,omitempty"`
-	Commit               string `json:"commit,omitempty"`
-	// helmVersionCommits records hash of each helm template version
+	LastRefreshTimestamp string             `json:"lastRefreshTimestamp,omitempty"`
+	Commit               string             `json:"commit,omitempty"`
+	Conditions           []CatalogCondition `json:"conditions,omitempty"`
+
+	// Deprecated: should no longer be in use. If a Catalog CR is encountered with this field
+	// populated, it will be set to nil.
 	HelmVersionCommits map[string]VersionCommits `json:"helmVersionCommits,omitempty"`
```

Comment:
@StrongMonkey recommended that we remove these deprecation notices since the `status` is not strictly part of the user-consumable API (`spec`). Leaving this open for reviewer's thoughts.

Reply:
Spoke to @dramich: we agree that these are fine to leave.